src/awx_plugins/credentials/plugins.py

```diff
@@ -8,6 +8,16 @@
     gettext_noop,
 )
 
+from awx_plugins.credentials.injectors import (
```

Comment:
Always use relative imports for local modules:
```suggestion
from .injectors import (
```